tests/acceptance/10_files/can_set_sticky_bits_without_root_owning_directory.cf

```diff
@@ -0,0 +1,80 @@
+#######################################################
+#
+# Acceptance test for issue #2745
+# Ensure that cfengine creates a
+# directory not owned by root and the setgid
+# is set on the directory
+#
+#######################################################
+
+body common control
+{
+      inputs => { "../default.cf.sub" };
+      bundlesequence  => { default("$(this.promise_filename)") };
+      version => "1.0";
+}
+
+
+#######################################################
+
+bundle agent init
+{
+
+      #######################################
+      #Clean up test directory before testing
+      #######################################
+
+  files:
+      "$(G.testdir)/test_sgid"
+        delete => tidy;
+
+}
```

Comment:
Note that these section are never necessary when running under `testall`, because it always starts with a clean slate. It would be nice not to clutter every test with this.